File: examples/python/usage_example.py
import qoery

configuration = qoery.Configuration()
configuration.api_key['ApiKeyAuth'] = 'YOUR_API_KEY'

with qoery.ApiClient(configuration) as api_client:
    api = qoery.UsageApi(api_client)
    usage = api.get_usage_stats()
    print(f"Plan: {usage.plan}, Usage: {usage.total_usage}")

File: examples/js/usage_example.js
const Qoery = require('qoery');

const client = Qoery.ApiClient.instance;
client.authentications['ApiKeyAuth'].apiKey = 'YOUR_API_KEY';

const api = new Qoery.UsageApi();

api.getUsageStats({}, (error, usage) => {
  if (error) throw error;
  console.log(`Plan: ${usage.plan}, Usage: ${usage.total_usage}`);
});
